target/linux/bcm53xx/config-6.6

```diff
@@ -183,6 +183,12 @@ CONFIG_HW_RANDOM=y
 CONFIG_HW_RANDOM_BCM2835=y
 CONFIG_HZ_FIXED=0
 CONFIG_HZ_PERIODIC=y
+CONFIG_I2C=y
+CONFIG_I2C_BOARDINFO=y
+CONFIG_I2C_BCM_IPROC=y
+CONFIG_I2C_CHARDEV=y
+CONFIG_I2C_HELPER_AUTO=y
+# CONFIG_I2C_SLAVE_TESTUNIT is not set
```

Comment:
Why do you need `CONFIG_I2C_BOARDINFO`, `CONFIG_I2C_CHARDEV`, and `CONFIG_I2C_HELPER_AUTO`?

Reply:
Probably leftovers from @clayface work, I'll double-check if that's still needed.

Reply:
All these can be removed. `CONFIG_I2C_BOARDINFO` is `default y` and I noticed it's enabled on all other targets, BTW. But even without these, EEPROM probes correctly and network interfaces can be set up with right MAC even at preboot. So - removed.